src/Chapter14/Listing14.11.UsingTheAssignmentOperationRatherThanPlusEquals.cs

```diff
@@ -14,24 +13,26 @@ public static void Main()
             Heater heater = new Heater(60);
             Cooler cooler = new Cooler(80);
 
-            // Note: Use new Action(cooler.OnTemperatureChanged)
-            // for C# 1.0 syntax
             thermostat.OnTemperatureChange =
                 heater.OnTemperatureChanged;
 
             // Bug: Assignment operator overrides 
             // previous assignment
+            #region HIGHLIGHT
             thermostat.OnTemperatureChange = 
                 cooler.OnTemperatureChanged;
+            #endregion HIGHLIGHT
 
             Console.Write("Enter temperature: ");
             string? temperature = Console.ReadLine();
             if (!int.TryParse(temperature, out int currentTemperature))
             {
-                Console.WriteLine($"'{temperature}' is not a valid integer.");
+                Console.WriteLine
+                    ($"'{temperature}' is not a valid integer.");
```

Comment:
Same comment here as earlier, maybe this should be made into one line? Thoughts anyone?